feat: confirmed & pending state in the highest layers (#119)
RequestNetwork/requestNetwork: packages/request-client.js/README.md

  meta, // see "Metadata of a request"
  balance,
  contentData,
  pending, // see "Pending state of a request"
}
*/
```

`requestData.request`: [IRequestData](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/request-client.js/src/types.ts#L17)

#### Pending state of a request

> Requests has a `pending` attribute containing all the change that are in transactions still pending:

Reviewer: ```suggestion
Requests have a `pending` attribute containing all the changes from transactions still pending.
```
